### ISSUE TITLE
overuse of aria-label on Dashboard

### ISSUE THREAD

**Issue by VlastiBroucek** [emsesp/EMS-ESP32#2710]
### DESCRIPTION

`aria-label` should be used only behind symbols, not readable texts. Using it in places where there is a visible text is discouraged due to possibility of confusing screen readers and other assistive technologies.

<img width="1025" height="343" alt="Image" src="https://github.com/user-attachments/assets/dfe1b87d-4a8f-440a-880a-a8454456c73e" />

It seems to be used correctly, although not consistently on other pages - e.g. pencil indicating editable item does not have `aria-label`. 

### REQUESTED INFORMATION

_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [X] Searched the issue in [issues](https://github.com/emsesp/EMS-ESP32/issues)
- [X] Searched the issue in [discussions](https://github.com/emsesp/EMS-ESP32/discussions)
- [X] Searched the issue in the [docs](https://docs.emsesp.org/Troubleshooting/)
- [ ] Searched the issue in the [chat](https://discord.gg/3J3GgnzpyT)
- [ ] Provide the System information in the area below, taken from `http://<IP>/api/system`

```json
  Paste System information here....


```

### TO REPRODUCE

Just hover over the text.

### EXPECTED BEHAVIOUR

If used, it should be used consistently and according to guidelines (https://www.w3.org/WAI/standards-guidelines/aria/)

### SCREENSHOTS

_If applicable, add screenshots to help explain your issue._

### ADDITIONAL CONTEXT

_Add any other context about the issue here._

**(Please remember to close the issue when it has been addressed)**

**Comment by proddy** [comment]
ok, I'll do a clean sweep through the webcode and see how much extra code would be generated,

**Comment by proddy** [comment]
> But it would be good to remove them from plain texts as in the picture in OP. They should not be there...

not sure i understand, can you give an example? what is OP?

**Comment by proddy** [comment]
> > > But it would be good to remove them from plain texts as in the picture in OP. They should not be there...
> > 
> > 
> > not sure i understand, can you give an example? what is OP? Do you mean the tool tips? or the text in the table. If I run the browser DevTools->Lighthouse->Accessability it looks ok with a score of 94% in the dashboard page
> 
> OP = original post... The picture at the start of this issue shows where the aria-label is but should not be. None of those values should gave it.

ah ok. I see that the entity value has an aria-label, which was automatically added by the HTML `<span>` tag. I'll remove it. Weirdly, you spotted it, as, according to the internet, it says "The aria-label attribute on `<span>` elements is often ignored by screen readers because <span> is a non-interactive element without semantic meaning"
